Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -615,3 +611,15 @@ void Renderer::Swap(u32 xfbAddr, u32 fbWidth, u32 fbStride, u32 fbHeight, const
 	XFBWrited = false;
 }
 
+void Renderer::FlipImageData(u8* data, int w, int h, int pixel_width)
```

Comment:
This code was already there. It's likely better to do it in this way, but I'm fine with just moving it as-it-is.